notebooks/new-31-js-language-overview.md

```diff
@@ -455,20 +455,31 @@ vector.display()
 
 ```javascript cell_style="split"
 class Temperature {
-    constructor(kelvin) {
-        this.kelvin = kelvin; // Call "set kelvin(kelvin)"
+    constructor(temp_value) {
```

Comment:
if we want to go down this road, I suggest we call the parameter more simply `temperature`

Reply:
yes !  and 'temp' could be understood as 'temporary'... 
done